### PR TITLE
Task-31-Help-Router-Front-End

### PR DESCRIPTION
**Pull request by slavasgod**

### CODE REVIEW

**Review by OlegPanasyuk — COMMENTED** [review]
The Review is finished. I like your implementation of layouts.

File: src/components/help/gettingStarted/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import '../../../scss/shared/colors.scss';
+@import '../../../scss/shared/fonts.scss';
+
+.help_page {
```
Comment (Pahmurny-itechart) [inline src/components/help/gettingStarted/styles.module.scss]: why are you duplicating this?

File: src/components/help/troubleshooting/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import '../../../scss/shared/colors.scss';
+@import '../../../scss/shared/fonts.scss';
+
+.help_page {
```
Comment (Pahmurny-itechart) [inline src/components/help/troubleshooting/styles.module.scss]: Same here. You could reuse this class from common module

File: src/components/help/index.js [hunk 3]
```diff
@@ -12,37 +13,37 @@ import styles from './styles.module.scss';
 export const Help = () => {
   return (
     <section className={styles.cards_wrapper}>
-      <a href="/" className={styles.card_link}>
+      <Link to="/getting_started" className={styles.card_link}>
         <div className={styles.card_wrapper}>
           <FaClipboardList />
           <h2>Getting Started</h2>
```
Comment (universezzz) [inline src/components/help/index.js]: It's not a good case to use `<h2>` more than 2 times in app, pls rewrite it to `<h3>`
`<h1>` - 1 time
`<h2>` - 2 time
h3,h4,h5,h6 - it doesn't matter
If this is just a mock component, and you will rewrite this code later, ok.

File: src/components/help/index.js [hunk 4]
```diff
@@ -12,37 +13,37 @@ import styles from './styles.module.scss';
 export const Help = () => {
   return (
     <section className={styles.cards_wrapper}>
```
Comment (universezzz) [inline src/components/help/index.js:10]: There is a good practise, to mark tags `<article>` `<section> ` with `<h1>...<h6>`.
In `<h>` you should write description of the block and then use css property `display: none` if this `<h>` isn't needed in the design of the project

File: src/components/layaout/index.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import MainPage from '../../containers/mainPage';
+import { Help } from '../help';
+
+export const Layaout = () => {
```
Comment (universezzz) [inline src/components/layaout/index.js]: Pls, rewrite to the 'Layout'

**Review by universezzz — COMMENTED** [review]
<3

**Pull request merged**